build_tools/kickstarts/centos-7-adb-vagrant.ks

```diff
@@ -128,67 +128,6 @@ echo "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4
-NiAtcGFzc291dCBmaWxlOiRwYXNzZmlsZSAtb3V0IGNhLWtleS5wZW0gMjA0OApvcGVuc3NsIHJl
-cSAtbmV3IC14NTA5IC1wYXNzaW4gZmlsZTokcGFzc2ZpbGUgLWRheXMgMzY1IC1rZXkgY2Eta2V5
-LnBlbSAtc2hhMjU2IC1vdXQgY2EucGVtIC1zdWJqICIvQz0vU1Q9L0w9L089L09VPS9DTj1leGFt
-cGxlLmNvbSIKCiMgR2VuZXJhdGUgU2VydmVyIEtleSBhbmQgU2lnbiBpdApvcGVuc3NsIGdlbnJz
-YSAtb3V0IHNlcnZlci1rZXkucGVtIDIwNDgKb3BlbnNzbCByZXEgLXN1YmogIi9DTj1leGFtcGxl
-LmNvbSIgLW5ldyAta2V5IHNlcnZlci1rZXkucGVtIC1vdXQgc2VydmVyLmNzcgojIEFsbG93IGZy
-b20gcm91dGFibGUgbG9jYWwgSVAKZXh0aXA9YGlwIGFkZHIgc2hvdyBldGgxIDI+L2Rldi9udWxs
-IHwgYXdrICdOUj09MyB7cHJpbnQgJDJ9JyB8IGN1dCAtZjEgLWRcL2AKaWYgW1sgLXogIiRleHRp
-cCIgXV07IHRoZW4KICAgICAgZXh0aXA9YGlwIHJvdXRlIGdldCA4LjguOC44IHwgYXdrICdOUj09
-MSB7cHJpbnQgJE5GfSdgCmZpCmV4dGlwZmlsZT1leHRmaWxlLmNuZgplY2hvIHN1YmplY3RBbHRO
-YW1lID0gSVA6JGV4dGlwID4gJGV4dGlwZmlsZQpvcGVuc3NsIHg1MDkgLXJlcSAtZGF5cyAzNjUg
-LWluIHNlcnZlci5jc3IgLUNBIGNhLnBlbSAtQ0FrZXkgY2Eta2V5LnBlbSAtQ0FjcmVhdGVzZXJp
-YWwgLW91dCBzZXJ2ZXItY2VydC5wZW0gLXBhc3NpbiBmaWxlOiRwYXNzZmlsZSAtZXh0ZmlsZSAk
-ZXh0aXBmaWxlCgojIEdlbmVyYXRlIHRoZSBDbGllbnQgS2V5IGFuZCBTaWduIGl0Cm9wZW5zc2wg
-Z2VucnNhIC1vdXQga2V5LnBlbSAyMDQ4Cm9wZW5zc2wgcmVxIC1zdWJqICcvQ049Y2xpZW50JyAt
-bmV3IC1rZXkga2V5LnBlbSAtb3V0IGNsaWVudC5jc3IKZXh0ZmlsZT10bXAuZXh0CmVjaG8gZXh0
-ZW5kZWRLZXlVc2FnZSA9IGNsaWVudEF1dGggPiAkZXh0ZmlsZQpvcGVuc3NsIHg1MDkgLXJlcSAt
-ZGF5cyAzNjUgLWluIGNsaWVudC5jc3IgLUNBIGNhLnBlbSAtQ0FrZXkgY2Eta2V5LnBlbSAtQ0Fj
-cmVhdGVzZXJpYWwgLW91dCBjZXJ0LnBlbSAtZXh0ZmlsZSAkZXh0ZmlsZSAtcGFzc2luIGZpbGU6
-JHBhc3NmaWxlCgojIENsZWFuIHVwCgojIHNldCB0aGUgY2VydCBwYXRoIGFzIGNvbmZpZ3VyZWQg
-aW4gL2V0Yy9zeXNjb25maWcvZG9ja2VyCgojIyBNb3ZlIGZpbGVzIGludG8gcGxhY2UKbXYgY2Eu
-cGVtICRET0NLRVJfQ0VSVF9QQVRICm12IHNlcnZlci1jZXJ0LnBlbSAkRE9DS0VSX0NFUlRfUEFU
-SAptdiBzZXJ2ZXIta2V5LnBlbSAkRE9DS0VSX0NFUlRfUEFUSAoKIyBzaW5jZSB0aGUgZGVmYXVs
-dCB1c2VyIGlzIHZhZ3JhbnQgYW5kIGl0IGNhbiBydW4gZG9ja2VyIHdpdGhvdXQgc3VkbwpDTElF
-TlRfU0lERV9DRVJUX1BBVEg9L2hvbWUvdmFncmFudC8uZG9ja2VyCgpta2RpciAtcCAkQ0xJRU5U
-X1NJREVfQ0VSVF9QQVRICmNwICRET0NLRVJfQ0VSVF9QQVRIL2NhLnBlbSAkQ0xJRU5UX1NJREVf
-Q0VSVF9QQVRICm12IGNlcnQucGVtIGtleS5wZW0gJENMSUVOVF9TSURFX0NFUlRfUEFUSAoKY2hv
-d24gdmFncmFudDp2YWdyYW50ICRDTElFTlRfU0lERV9DRVJUX1BBVEgKCmNobW9kIDA0NDQgJENM
-SUVOVF9TSURFX0NFUlRfUEFUSC9jYS5wZW0KY2htb2QgMDQ0NCAkQ0xJRU5UX1NJREVfQ0VSVF9Q
-QVRIL2NlcnQucGVtCmNobW9kIDA0NDQgJENMSUVOVF9TSURFX0NFUlRfUEFUSC9rZXkucGVtCmNo
-b3duIHZhZ3JhbnQ6dmFncmFudCAkQ0xJRU5UX1NJREVfQ0VSVF9QQVRIL2NhLnBlbQpjaG93biB2
-YWdyYW50OnZhZ3JhbnQgJENMSUVOVF9TSURFX0NFUlRfUEFUSC9jZXJ0LnBlbQpjaG93biB2YWdy
-YW50OnZhZ3JhbnQgJENMSUVOVF9TSURFX0NFUlRfUEFUSC9rZXkucGVtCgpjaG1vZCAtdiAwNDAw
-ICRET0NLRVJfQ0VSVF9QQVRIL2NhLnBlbSAkRE9DS0VSX0NFUlRfUEFUSC9zZXJ2ZXItY2VydC5w
-ZW0gJERPQ0tFUl9DRVJUX1BBVEgvc2VydmVyLWtleS5wZW0KCiMjIFJlbW92ZSByZW1haW5pbmcg
-ZmlsZXMKY2QKcm0gLXJmICRkaXIKCiMgRW5kIG9mIGNlcnRzIHBhaXIgZ2VuZXJhdGlvbiBzdGVw
-cyBmb3IgVExTIGVuYWJsZWQgZG9ja2VyIGRhZW1vbgo=
-EOF
-
-mkdir -p /opt/adb
-base64 -d < cert-gen.sh.base64 > cert-gen.sh
-mv cert-gen.sh /opt/adb/
-chmod u+x /opt/adb/cert-gen.sh
```

Comment:
@praveenkumar this is interesting. We have not put anything specific for the permissions in the adb-utils spec file. Wondering how it got +x?